plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1029,6 +1032,181 @@ public void testMultiIndexAlias()
                 "SELECT (SELECT count(*) FROM region) + (SELECT count(*) FROM nation)");
     }
 
+    @Test
+    public void testDescribeMultiIndexAlias()
```

Comment:
On this test, could you add one index that matches the `racks` index metadata (e.g. `racks2`) and then make a fourth index (e.g. `snacks`) that brings a new `snackid` field that needs to get merged. I just notice that only two indices are used across the board, and I would like at least have one test that has to merge more than two indices.

Reply:
Done. Also added a test `testSelectMultiIndexAlias` to unit test select queries on alias table.